src/mkdocs_gallery/gen_single.py

```diff
@@ -974,6 +974,12 @@ def generate_file_md(script: GalleryScript, seen_backrefs=None) -> GalleryScript
                 script.gallery_conf['stale_examples'].append(script.dwnld_py_file.as_posix())
 
         if skip_and_return:
+            # If expected to fail, let's assume it did when executed previously
+            if script.src_py_file in script.gallery_conf['expected_failing_examples']:
+                script.gallery_conf['failing_examples'][script.src_py_file] = (
+                    "Due to MD5 check, script has not been actually executed - "
+                    "Assumed it failed as expected during previous execution."
+                )
```

Comment:
I would be curious to see how this is done in `sphinx-gallery`. Maybe there is an opportunity to suggest them a PR ?

Reply:
Indeed I did not check `sphinx-gallery` before implementing the proposed solution.

https://github.com/sphinx-gallery/sphinx-gallery/blob/26fc508bac5459c3d6daf5db64cf5d45cda53b7d/sphinx_gallery/gen_rst.py#L961-L969

It seems they handle this by using the `gallery_conf['stale_examples']` list instead of the `gallery_conf['failing_examples']` dict. I'm not familiar enough with the global process to evaluate the best approach. It seemed natural to me to use the 'failing_examples' entry, but maybe we should give it a try.

Reply:
oh-oh ! it seems that there is an "as_posix" here ! :)